Latex/vorlage/Doku-Test.tex

```diff
@@ -54,43 +54,29 @@ \section{Programmcode}
 \vgcaption{https://link-wo-es-den-code-gibt.de}{06.07.2022}
 \section{Ordnerstruktur}
     \verzeichnis{%
-          .1 \dtfolder Vorlage-Latex.
-            .2 \dtfile HINWEISE.md.
-            .2 \dtfile LICENSE.
-            .2 \dtfile README.md.
-            .2 \dtfile sortieren.py.
-            .2 \dtfolder Latex.
-              .3 \dtfolder bilder.
-                .4 \dtfile firmenlogo.svg.
-              .3 \dtfolder inhalt.
-                .4 \dtfile Abkürzungen.tex.
-                .4 \dtfile Anhang.tex.
-                .4 \dtfile Kapitel1.tex.
-              .3 \dtfolder light.
-                .4 \dtfile main.tex.
-                .4 \dtfile README.md.
-                .4 \dtfile vorlage\_light.tex.
-              .3 \dtfile literatur.bib.
-              .3 \dtfile main.tex.
-              .3 \dtfile metadaten.sty.
-              .3 \dtfolder vorlage.
-                .4 \dtfile Abstract\_Freigabeerklärung\_Bachelorthesis.tex.
-                .4 \dtfile ba-gc-logo.png.
-                .4 \dtfile ba-gc-logo.svg.
-                .4 \dtfile dirtree\_file.svg.
-                .4 \dtfile dirtree\_folder.svg.
-                .4 \dtfile Erklärung\_1Autor.tex.
-                .4 \dtfile Erklärung\_2Autoren.tex.
-                .4 \dtfile Erklärung\_3Autoren.tex.
-                .4 \dtfile Erklärung\_4Autoren.tex.
-                .4 \dtfile Titelseite\_1Autor.tex.
-                .4 \dtfile Titelseite\_2Autoren.tex.
-                .4 \dtfile Titelseite\_3Autoren.tex.
-                .4 \dtfile Titelseite\_4Autoren.tex.
-                .4 \dtfile vorlage.tex.
+          .1 \dtfolder Vorlage-Latex. .2 \dtfile HINWEISE.md.
+          .2 \dtfile LICENSE.
+          .2 \dtfile README.md.
+          .2 \dtfile sortieren.py.
+          .2 \dtfolder Latex.
+            .3 \dtfolder bilder.
+              .4 \dtfile firmenlogo.svg.
+            .3 \dtfolder inhalt.
+              .4 \dtfile Abkürzungen.tex.
+              .4 \dtfile Anhang.tex.
+            .3 \dtfolder light.
+              .4 \dtfile main.tex.
+              .4 \dtfile README.md.
+              .4 \dtfile vorlage\_light.tex.
+            .3 \dtfile literatur.bib.
+            .3 \dtfile main.tex.
+            .3 \dtfile main\_abstract.tex.
+            .3 \dtfile metadaten.sty.
     }{Ein Verzeichnis-Baum}{beispielbaum}
 
     \section{Formeln}
+    Formeln werden mittels \striche{\textbackslash formula} in die Arbeit eingebunden.
```

Comment:
ich dachte immer, n leerzeichen nach nem command macht ein leerzeichen (im gegensatz zu `\textbackslash{}formula`)

ist aber hier gar nicht der fall (hat gebaut kein leerzeichen) - wieso? :D

Reply:
is ja richtig so was is das problem?:D

Reply:
die kugel der verwirrung